discovery/src/main/java/com/ibm/watson/developer_cloud/discovery/v1/query/Aggregation.java

```diff
@@ -11,7 +11,7 @@
  * specific language governing permissions and limitations under the License.
  */
 
-package com.ibm.watson.developer_cloud.discovery.v1.model.query;
+package com.ibm.watson.developer_cloud.discovery.v1.query;
```

Comment:
I thought these were being used in the tests, but on closer inspection I see that the tests only use the enum classes for query generation.  I'll remove the rest.